contrib/scalar/scalar.c

```diff
@@ -178,6 +196,27 @@ static int set_recommended_config(int reconfigure)
 		}
 	}
 
+	/*
+	 * The `log.excludeDecoration` setting is special because we want to
+	 * set multiple values.
+	 */
+	if (get_config_get_string("log.excludeDecoration", &value)) {
+		trace2_data_string("scalar", the_repository,
+				   "log.excludeDecoration", "created");
+		if (git_config_set_multivar_gently("log.excludeDecoration",
+						   "refs/scalar/*",
+						   CONFIG_REGEX_NONE, 0) ||
```

Comment:
This one is leftover from a previous fetch step. Perhaps make a note that we don't need to send this one upstream? Or, maybe we should, anyway. One way to (in a future change) make this optional would be to see if any `refs/scalar/*` ref exists. Don't worry about it now.

Reply:
Okay, I won't, then ;-)